internal/worker/bootstrap/manifold.go

```diff
@@ -81,6 +81,12 @@ type NetworkService interface {
 	ReloadSpaces(ctx context.Context) error
 }
 
+// BakeryConfigService is the interfcae that is used to initialise the
```

Comment:
**todo(typo):** `interfcae` => `interface`. And by the way, I don't think that stating it's an interface has any value. What is interesting is what is the BakeryConfigService concept. A BakeryConfigService is a singleton (probably... maybe not... But since it take a context and return an error, it work on some context-scoped data, or global one... So it's probably a singleton, conceptually ? anyway :D ) which is used to.... blablaba.

Reply:
`interface` is used in doc-strings for other service interfaces in this file, but I agree, on inspection it's pretty bad

Re-worded